frontend/src/lib/layout/DetailItem.svelte

```diff
@@ -0,0 +1,43 @@
+<script lang="ts">
+  import CopyToClipboardButton from '$lib/components/CopyToClipboardButton.svelte';
+  import EditableText from '$lib/components/EditableText.svelte';
+  import type { ErrorMessage } from '$lib/forms';
+  import t, { type I18nKey } from '$lib/i18n';
+
+  export let title: I18nKey;
+  export let text: string | null | undefined;
+  export let copyToClipboard = false;
+  export let editable = false;
+
+  // Items below only used when editable = true
```

Comment:
not sure I love this. This component does 2 things, it's either an editor or a display, will we ever need to switch between the 2?